contents/docs/data/cohorts.mdx

```diff
@@ -22,97 +22,83 @@ availability:
             enterprise: true
 ---
 
-Cohorts are PostHog's way of letting you easily create a list of users who have something in common.
+Cohorts enable you to easily create a list of users who have something in common. You can use the cohorts to view a list of users who all did the same action or have the same property.
 
-You can use the cohorts to:
+Here are a few examples of the cohorts you can create:
 
--   View a list of users who all did the same action or have the same property. A few examples of how you might use this are:
-    -   "Which users used the app today?"
-    -   "Which users work at the same company?"
-    -   "Who used the app once but never came back?"
--   Segment other statistics. For example, to view user paths or action trends. This let's you understand things like:
-    -   "Are people at company X using the app much?"
-    -   "Is the number of Daily Active Users going up or down over time?"
+- Users who work at the same company.
+- Users who used your app in the last week.
+- Users who have churned from your product.
```

Comment:
```suggestion
- work at the same company.
- used your app in the last week.
- churned from your product.
- signed up recently.
- viewed the signup page, but didn't convert.
```

Reply:
Hmm I think it's clearer to have "Users who", but I like your additional examples. I'll add them